connect/cli/plugins/product/sync/actions.py

```diff
@@ -74,7 +75,9 @@ def sync(self):  # noqa: CCR001
 
             if data.action == 'create':
                 try:
-                    original_action = list(filter(lambda x: x['action'] == data.id, actions))
+                    def _check_if_matches(data, x):
```

Comment:
this change is due to the bump of flake8 and related ?

Reply:
Yes, it's B023: Function definition does not bind loop variable 'data'. Because we are accessing the variable data from inside the lamba function.